js/popup2.js

```diff
@@ -161,38 +159,41 @@ function doBadgePrintDownload(attendeeData) {
         //console.log(tabs[0].id);
         chrome.tabs.sendMessage(tabs[0].id, {action: "Increment Badge Count"}, function() {
             console.log("Should have saved now");
+            window.close();
+            // Window close seems to work more consistently here
         } );
     });
     //window.confirm("Printing Completed");
     //window.close();
 }
 
-function saveTextAsFile(badgeNumText, badgeNameText, background, tix, printNumber)
+function saveTextAsFile(accountId, attendeeId, badgeNameText, background, tix, printNumber)
 {
     // This is from
     // http://thiscouldbebetter.wordpress.com/2012/12/18/loading-editing-and-saving-a-text-file-in-html5-using-javascrip/
 
     //var textToWrite = "THIS IS SAMPLE TEXT TO WRITE TO A FILE";
     //var textFileAsBlob = new Blob([textToWrite], {type:'text/plain'});
-    var textHeader = '"Badge Type","Badge File BackGround","Badge Number","Badge Name","Workstation ID","Print Number"';
+    var textHeader = '"Badge Type","Badge File BackGround","Account ID","Attendee ID","Badge Name","Workstation ID","Print Number"';
     var text = new Array();
     text[0] = tix;
     text[1] = background;
-    text[2] = badgeNumText;
-    text[3] = badgeNameText;
+    text[2] = accountId;

```

Comment:
Is this where it is important?  I thought the format of this CSV file was very ridged. You are adding a field. This ok with the badge printer?

Reply:
Yeah, we'll have to make corresponding edits to the printing script, but this was something Thomas requested.

Reply:
ok cool, thanks!